huaweicloud/services/iotda/resource_huaweicloud_iotda_device.go

```diff
@@ -0,0 +1,450 @@
+package iotda
+
+import (
+	"context"
+	"fmt"
+	"regexp"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	v5 "github.com/huaweicloud/huaweicloud-sdk-go-v3/services/iotda/v5"
```

Comment:
the package name already is **v5**, alias is unnecessary.

Reply:
It's auto-format by vscode.